docs-site/content/docs/getting-started/tour/index.md

```diff
@@ -140,6 +142,22 @@ Done! enjoy your ride with `loco` 🚂
 
 Your generated app contains a fully working authentication suite, based on JWTs.
 
+To authenticate, you will need a running redis server. 
```

Comment:
To make the tutorial with less friction, we can switch the worker mode to inprocess background: `BackgroundAsync`.
This removes the need for Redis.
The need for workers is for background sending of registration emails etc.


Reminder, the worker config:

```
 # Worker Configuration
workers:
  # specifies the worker mode. Options:
  #   - BackgroundQueue - Workers operate asynchronously in the background, processing queued.
  #   - ForegroundBlocking - Workers operate in the foreground and block until tasks are completed.
  #   - BackgroundAsync - Workers operate asynchronously in the background, processing tasks with async capabilities.
  mode: BackgroundQueue

```

Reply:
So how to go about this? Change the Saas default to BackgroundAsync or have them manually change it in the Tour?

Also, if BackgroundAsync is inprocess, then the config documentation should be clear about which ones require external database and which ones are done inprocess so I'll update that as well I guess

Reply:
Good question. 
We're asking the user to do either of the two:

1. Configure and set up Redis
2. Change a setting in their config.yaml

What do you think makes the least friction and mental effort? (we want to remember that this is kind of a "quick tour")

Reply:
If we are talking about most easiest, since you are already setting up docker for Postgres and are already on the shell, copy and pasting running Redis is probably easier than going through a config file. Unless you are setting that config file via shell as well. I mean there is nothing less mental effort than copy and paste

On the up side, changing the config file gets the user more aware of the config files existance

Reply:
yes, that sounds convincing. so let's go with bringing up another Docker.